pkg/operator/ceph/provisioner/provisioner.go

```diff
@@ -110,8 +113,10 @@ func (p *RookVolumeProvisioner) Provision(options controller.VolumeOptions) (*v1
 		Spec: v1.PersistentVolumeSpec{
 			PersistentVolumeReclaimPolicy: options.PersistentVolumeReclaimPolicy,
 			AccessModes:                   options.PVC.Spec.AccessModes,
+			// since we can guarantee the size of the volume image generated have to be in `MB` boundary, so
+			// we can convert it to `MB` unit safely here
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): capacity,
+				v1.ResourceName(v1.ResourceStorage): resource.MustParse(fmt.Sprintf("%dMi", blockImage.Size/sizeMB)),
```

Comment:
Use `ParseQuantity` instead of `MustParse`, because `MustParse` will panic, but we want to be able to handle the error.

Reply:
Replace the `MustParse` with `ParseQuantity` and add error handling in our code. @galexrt